orchestrator.Jenkinsfile

```diff
@@ -280,40 +301,69 @@ String projectVersion(project) {
+global.event-service.dir=${versions['components/event-service'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.alpine-net.version=${versions['tools/alpine-net']}
+global.alpine-net.dir=${versions['tools/alpine-net'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.watch-pods.version=${versions['tools/watch-pods']}
+global.watch-pods.dir=${versions['tools/watch-pods'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.stability-checker.version=${versions['tools/stability-checker']}
+global.stability-checker.dir=${versions['tools/stability-checker'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.etcd-backup.version=${versions['tools/etcd-backup']}
+global.etcd-backup.dir=${versions['tools/etcd-backup'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.etcd-tls-setup.version=${versions['tools/etcd-tls-setup']}
+global.etcd-tls-setup.dir=${versions['tools/etcd-tls-setup'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.test-logging-monitoring.version=${versions['tests/test-logging-monitoring']}
+global.test-logging-monitoring.dir=${versions['tests/test-logging-monitoring'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.acceptance-tests.version=${versions['tests/acceptance']}
+global.acceptance-tests.dir=${versions['tests/acceptance'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.ui-api-layer-acceptance-tests.version=${versions['tests/ui-api-layer-acceptance-tests']}
+global.ui-api-layer-acceptance-tests.dir=${versions['tests/ui-api-layer-acceptance-tests'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.gateway-tests.version=${versions['tests/gateway-tests']}
+global.gateway-tests.dir=${versions['tests/gateway-tests'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.test-environments.version=${versions['tests/test-environments']}
+global.test-environments.dir=${versions['tests/test-environments'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.kubeless-test-client.version=${versions['tests/kubeless-test-client']}
+global.kubeless-test-client.dir=${versions['tests/kubeless-test-client'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.api-controller-acceptance-tests.version=${versions['tests/api-controller-acceptance-tests']}
+global.api-controller-acceptance-tests.dir=${versions['tests/api-controller-acceptance-tests'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.connector-service-tests.version=${versions['tests/connector-service-tests']}
+global.connector-service-tests.dir=${versions['tests/connector-service-tests'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.metadata-service-tests.version=${versions['tests/metadata-service-tests']}
+global.metadata-service-tests.dir=${versions['tests/metadata-service-tests'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.event-bus-tests.version=${versions['tests/event-bus']}
+global.event-bus-tests.dir=${versions['tests/event-bus'] == env.BRANCH_NAME ? 'pr' : 'develop'}
+global.test-logging.version=${versions['tests/logging']}
+global.test-logging.dir=${versions['test-logging'] == env.BRANCH_NAME ? 'pr' : 'develop'}
```

Comment:
Not sure if `versions['test-logging']` if this is right

Reply:
Yes, that was a mistake...
Thanks for catching that one! Fixed.